### PR TITLE
Electron-266 (Custom title bar for Windows 10)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Custom titlebar design for Windows 10 [ELECTRON-266](https://perzoinc.atlassian.net/browse/ELECTRON-266)

#### Before
titlebar_before

#### After
titlebar_after

## Learning
N/A

#### Blog Posts
N/A

## Open Questions if any and Todos
- [ ] Unit-Tests
- [ ] Documentation
- [ ] Automation-Tests
When solved, check the box and explain the answer.

@VikasShashidhar & @VishwasShashidhar Please review. Thanks 😄

### CODE REVIEW

File: js/windowsTitleBar/index.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            this.window.unmaximize();
+        } else {
+            this.window.maximize();
+        }
+    }
+
+    /**
+     * Method that closes the browser window
+     */
+    closeButtonClick() {
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/index.js]: Please rename this method something better. Perhaps `CloseWindow`

File: js/windowsTitleBar/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
+#title-bar {
+    display: flex;
+    position: fixed;
+    background: #4a4a4a;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: RGBA please

File: js/windowsTitleBar/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 32px;
+    padding-left: 10px;
+    justify-content: center;
+    align-items: center;
+    margin-left: -10px;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Why use `margin-left` when you get full control over this element when you are making its position fixed?

File: js/windowsTitleBar/style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    width: 40px;
+    height: 32px;
+    background: none;
+    border: none;
+    border-image: initial;
+    display: inline-grid;
+    border-radius: 0;
+    padding: 11px;
+    box-sizing: border-box;
+    cursor: default;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: What does Slack do for this? Does it not use a pointer?

File: js/windowsTitleBar/style.css [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    align-items: center;
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+.title-bar-button-container {
+    justify-content: center;
+    align-items: center;
+    right: 0;
+    color: white;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: RGBA please

File: js/windowsTitleBar/style.css [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    -webkit-app-region: no-drag;
+    text-align: center;
+    width: 45px;
+    height: 32px;
+    background: #4a4a4a;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: RGBA please

File: js/windowsTitleBar/style.css [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    margin: 0;
+    box-sizing: border-box !important;
+    cursor: default;
+}
+
+.title-bar-button {
+    color: white;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Same here

File: js/windowsTitleBar/style.css [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    border: none;
+    border-image: initial;
+    display: inline-grid;
+    border-radius: 0;
+    padding: 10px 15px;
+    cursor: default;
+}
+
+.title-bar-button:hover {
+    background-color: #333333;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Same here

File: js/windowsTitleBar/style.css [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.title-bar-button:focus {
+    outline: none;
+}
+
+.title-bar-button-container:hover {
+    background-color: #333333;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Same here

File: js/windowsTitleBar/style.css [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.window-border {
+    border-left: 1px solid #4a4a4a;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Same here

File: js/windowsTitleBar/style.css [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+    border-right: 1px solid #4a4a4a;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Same here

File: js/windowsTitleBar/style.css [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.bottom-window-border {
+    position: fixed;
+    border-bottom: 1px solid #4a4a4a;
```
Comment (VikasShashidhar) [inline js/windowsTitleBar/style.css]: Same here

File: js/windowsTitleBar/style.css (hunk 4 above)
Comment (KiranNiranjan) [reply js/windowsTitleBar/style.css]: Both on Slack and native Windows it is default cursor style 😄

**Comment by VishwasShashidhar** [comment]
@VikasShashidhar Please complete your review so we can merge this. Thanks!

**Pull request merged**